.buildkite/scripts/common.sh

```diff
@@ -336,19 +399,90 @@ config_git() {
   fi
 }
 
+withModule() {
+  local module_path=$1
+  if [[ "$module_path" == *"x-pack/"* ]]; then
+    local pattern=("$XPACK_MODULE_PATTERN")
+  else
+    local pattern=("$OSS_MODULE_PATTERN")
+  fi
+  local module_name="${module_path#*module/}"
+  local module_path_transformed=$(echo "$module_path" | sed 's/\//\\\//g')
+  local module_path_exclussion="((?!^${module_path_transformed}\\/).)*\$"
+  local exclude=("^(${module_path_transformed}|((?!\\/module\\/).)*\$|.*\\.asciidoc|.*\\.png)")
+  if are_paths_changed "${pattern[@]}" && ! are_changed_only_paths "${exclude[@]}"; then
+    export MODULE=${module_name}
+  elif [ -d "${module_path}" ]; then
+    export MODULE="aws"                 # TODO: remove this line and uncomment the line below when the issue https://github.com/elastic/ingest-dev/issues/2993 is solved
+    # export MODULE=''                  # TODO: uncomment the line when the issue https://github.com/elastic/ingest-dev/issues/2993 is solved
+  fi
+  echo "MODULE=$MODULE"
+}
+
+terraformInit() {
+  local dir=$1
+  echo "Terraform Init on $dir"
+  pushd "${dir}" > /dev/null
+  terraform init
+  popd > /dev/null
+}
+
+withAWS() {
+  export AWS_ACCESS_KEY_ID=$BEATS_AWS_ACCESS_KEY
```

Comment:
Maybe a comment saying these env variables were set in the `pre-command` hook to be able to mask their values. Then it's required to create new set of env variables that are using the right variable name?